CHANGELOG.md

```diff
@@ -7,6 +7,10 @@
 - Fixed PageView telemetry data not being reported. 
 - Fixed Issue #526 (NPE in MapUtil.copy())
 
+## Version 2.0.0
+
+- Fix #506 Null Reference Check causing Null Pointer Exception in `TelemetryCorrelationUtils.java`
+
```

Comment:
There should be more items in this changelog for 2.0.0 - i believe this PR is missing the last commits from Stable branch